.github/workflows/pr-build-workflow.yml

```diff
@@ -30,8 +30,9 @@ jobs:
           arguments: check
 
       - name: Capture Test Results
+        if: ${{ failure() }}
         uses: actions/upload-artifact@v2
         with:
           name: test-results
-          path: spring-rabbit/build/reports/tests
+          path: 'spring-rabbit/build/reports/tests/**/*.*'
```

Comment:
Don't we need to gather all the test reports?
From all the modules in the project?

Reply:
Yes; I am testing that now on Kafka.